### PR TITLE
Keywords

### PR DESCRIPTION
**Pull request by MorganR**
My implementation changes for the new keywords

### CODE REVIEW

File: ptsrc/parser/scan.def [hunk 1]
```diff
@@ -1,113 +0,0 @@
-	{ Semantic Operations }
```
Comment (MorganR) [inline ptsrc/parser/scan.def:1]: File part of "make" output

File: ptsrc/parser/stdIdentifiers [hunk 2]
```diff
@@ -39,7 +41,7 @@ repeat
 
 % Predeclared type names
 integer
-char
+string
```
Comment (MorganR) [inline ptsrc/parser/stdIdentifiers:44]: This is an interesting one. It doesn't seem to be used anywhere else, so I haven't changed anything more yet. Will do some testing

Comment (derekshum) [reply ptsrc/parser/stdIdentifiers:44]: This and changing ' to " seem to be the only changes required for this.

Comment (MorganR) [reply ptsrc/parser/stdIdentifiers:44]: the ' to " is something you have done right?

**Review by MorganR — COMMENTED** [review]
whoops

**Comment by derekshum** [comment]
Looks good, other than some possible issues with the tokens at the top of scan.ssl and parser.pt (and their numbering), which will be resolved later.

**Pull request merged**